community/cypher/compatibility-suite/src/test/resources/cypher/db/cineast/params.json

```diff
@@ -1,9 +1,9 @@
 {
-  "user_csv" : "file:User.csv",
-  "person_csv" : "file:Person.csv",
-  "movie_csv" : "file:Movie.csv",
-  "acts_in_csv" : "file:ACTS_IN.csv",
-  "directed_csv" : "file:DIRECTED.csv",
-  "friend_csv" : "file:FRIEND.csv",
-  "rated_csv" : "file:RATED.csv"
+  "user_csv" : "file:///User.csv",
+  "person_csv" : "file:///Person.csv",
+  "movie_csv" : "file:///Movie.csv",
+  "acts_in_csv" : "file:///ACTS_IN.csv",
+  "directed_csv" : "file:///DIRECTED.csv",
+  "friend_csv" : "file:///FRIEND.csv",
+  "rated_csv" : "file:///RATED.csv"
```

Comment:
@davidegrohmann Yes - I did. One advantage to using `load_csv_file_url_root` is that the files become chrooted to a configured directory and thus the file URL becomes OS agnostic.